src/backend/routers/iep.ts

```diff
@@ -160,6 +173,19 @@ export const iep = router({
     .mutation(async (req) => {
       const { subgoal_id, para_ids, due_date, trial_count } = req.input;
 
+      const existingTasks = await req.ctx.db
+        .selectFrom("task")
+        .where("subgoal_id", "=", subgoal_id)
+        .where("assignee_id", "in", para_ids)
+        .selectAll()
+        .execute();
+
+      if (existingTasks.length > 0) {
```

Comment:
If any task for this benchmark already includes one or more of the selected para's, then throw the error